snmp/datadog_checks/snmp/config.py

```diff
@@ -355,3 +357,18 @@ def get_table_symbols(mib, table):
             all_oids.insert(0, oids)
 
         return all_oids, bulk_oids, parsed_metrics
+
+    def add_uptime_metric(self):
+        if self._uptime_metric_added:
+            return
+        uptime_oid = '1.3.6.1.2.1.1.3.0'
```

Comment:
Is it worth making this a constant, potentially with a link to some reference?

Reply:
I added a link, but I don't think we need a constant, we only use it here.